kratos/python/add_modeler_to_python.cpp

```diff
@@ -50,6 +50,20 @@ void GenerateMesh(Modeler& GM, ModelPart& model_part, const char* ElementName, c
 
 }
 
+void GeneratePartialModelPart(ConnectivityPreserveModeler& GM, ModelPart& origin_model_part, ModelPart& destination_model_part, const char* Name)
```

Comment:
Just for me to know, why `const char *name` instead of `const std::string`?

Reply:
No good reason, char* is a C string, std::string is a C++ one. It looks like Pybind can convert python strings to either? It might be a good idea to check, since using std::strings could potentially avoid a copy here.

Reply:
Actually I think that Codacy is complaining about this copy.